client/src/components/Main/index.js

```diff
@@ -12,8 +15,10 @@ const Wrapper = styled.div`
 const Main = () => {
   return (
     <Wrapper>
-      <Invitation isUnopened={true}></Invitation>
-      <ActionCard></ActionCard>
+      {/* <Invitation isUnopened={true}></Invitation>
```

Comment:
Can you revert your changes for this file (Main/index.js)? We don't need this code in the master branch since it was just used for development.